plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/actions/Action.java

```diff
@@ -15,8 +15,20 @@
  ********************************************************************************/
 package org.eclipse.glsp.server.actions;
 
+/**
+ * Java-implementation of the sprotty `Action` interface. An action is a declarative description of a behavior that
```

Comment:
```suggestion
 * Java-implementation of the `Action` interface. An action is a declarative description of a behavior that
```

I have the feeling to a user it is confusing to consider Action as a Sprotty concept, since it is used as a base mechansim for the entire GLS protocol too.

Reply:
I agree, however, I aligned this with the protocol.md file in eclipse-glsp/glsp. So we should probably change it there as well